syntax = "proto3";

package db_sql;

service IpDatabaseService {
  rpc getIp(IpRequest) returns (IpResponse) {}
}

message IpRequest {
    string nickname = 1;
}
  
message IpResponse {
    int32 user_id = 1;
    string ip = 2;
    string birth_date = 3;
    string name = 4;
    string additional_information = 5;
}